scripts/commands.js

```diff
@@ -118,6 +118,7 @@ program
 program
   .command('pull_l10n')
   .option('--extension <extension>', 'Scope this command to localize a Brave extension such as ethereum-remote-client')
+  .option('--grd_path <grd_path>', `Relative path to match end of full GRD path, e.g: '/generated_resources.grd'.`)
```

Comment:
Nit: elsewhere in this file we seem to be using 'text, e.g: "example"' quoting pattern.